content/post/2024-10-01-same-same-but-different-how-advanced-data-science-techniques-help-us-validate-drug-names/index.md

```diff
@@ -0,0 +1,628 @@
+---
+title: 'Same-Same, but Different: How Advanced Data Science Techniques Help Us Validate
+  Drug Names'
+authors: ["kamil-pytlak"]
+date: '2024-10-01'
+categories:
+  - machine learning
+  - R
+  - statistics
+  - text analysis
+tags:
+  - data visualization
+  - drug names
+  - eCRF
+  - data validation
+  - levenshtein distance
+  - NLP
+  - t-SNE
```

Comment:
What is the difference between tags and categories? I think only one of these is presented.